user_guide_src/source/database/events/001.php

```diff
@@ -2,9 +2,3 @@
 
 // In Config\Events.php
 Events::on('DBQuery', 'CodeIgniter\Debug\Toolbar\Collectors\Database::collect');
-
-// Collect the queries so something can be done with them later.
-public static function collect(CodeIgniter\Database\Query $query)
-{
-    static::$queries[] = $query;
-}
```

Comment:
I do not see any use of this example at all.
It is just code that is already there in `Config\Events` and in the collector.

Maybe completely remove it?

Reply:
This is an example of how to use the database event.
I hope sample code.
E.g. logging all queries to a log file.

Reply:
Yes, but the example is already implemented in CI, so I am not sure it is really helpful:
https://github.com/codeigniter4/CodeIgniter4/blob/7dc2ece32401ebde67122f7d2460efcaee7c352e/app/Config/Events.php#L45
https://github.com/codeigniter4/CodeIgniter4/blob/7dc2ece32401ebde67122f7d2460efcaee7c352e/system/Debug/Toolbar/Collectors/Database.php#L78-L103

Reply:
Okay, let's go with this for now.

Probably, adding something another sample code is better.